src/swap/types.ts

```diff
@@ -164,6 +164,9 @@ export type SwapButtonReact = {
 
 export type SwapContextType = {
   address?: Address; // Used to check if user is connected in SwapButton
+  config: {
+    maxSlippage: number; // Maximum acceptable slippage for a swap. (default: 10) This is as a percent, not basis points
+  };
```

Comment:
Synced with @alessey here. We think it's best to add config to the SwapContext to have these values easily accessible to multiple components.

Reply:
could extract this to a config type and use in both places

Reply:
good idea ✅